tests/performance/event_generator/README.md

```diff
@@ -0,0 +1,112 @@
+# Event generator system
+
+This repository contains tools and scripts designed to simulate log and file system events. These tools are ideal for performance and stability testing of monitoring systems.
+
+# Components
+
+- **Log Event Generator**: Simulates logging events at specified intervals and volumes. Can be configured for log rotation based on size constraints.
+- **File System Event Generator**: Simulates file creation, modification, and deletion events, useful for testing file integrity monitoring systems.
+
+# Usage
+
+To run the event generation system, you can use the `main.py` script with an associated configuration file(`config.yaml`). This setup allows you to define detailed parameters for the simulation of log and file events.
+
+```bash
+python3 main.py --config config.yaml
+```
+
+This command starts the event simulation as specified in config.yaml, handling both log generation and file system events based on the module configurations defined within the file.
+
+# Configuration
+
+Modify the config.yaml file to set the types of events, frequency, target paths for logs and file operations, among other parameters. An example configuration might look like this:
+
+## Common parameters
+
+- **module**: Specifies the type of event generator. Options are logcollector or syscheck.
+- **path**: The filesystem path where logs will be written or where file operations will occur.
+- **rate**: The rate at which events are generated per second.
+- **cleanup**: If true, the generated files or directories will be deleted after the event generation is complete.
+
+## Log event generator parameters
+
+- **operations**: The total number of events to generate before stopping.
+- **max_file_size**: The maximum file size before the log file is truncated. Value is in megabytes (MB).
+To disable the size limit, set max_file_size to null.
+- **template_path**: Path to a JSON template file for log formatting.
+
+## Syscheck event generator parameters
+
+- **num_files**: The number of files to create.
+- **num_modifications**: The number of times each file should be modified.
+
+## Example configuration
+
+```yaml
+files:
+  - module: logcollector
+    path: /path/to/logfile.log
+    operations: 100
+    rate: 2
+    max_file_size: 10
+    template_path: templates/syslog_template.json
+    cleanup: true
+
+  - module: syscheck
+    path: /path/to/test/directory
+    num_files: 50
+    num_modifications: 10
+    rate: 1
+    cleanup: true
```

Comment:
Now we are not able to specify selected fixed operations per second. We need to eliminate syscheck randomness without affecting functional requirements.

Reply:
Fixed here: https://github.com/wazuh/wazuh-qa/pull/5697/commits/376d046f24b879eaa8c5dc0b196dc7a352fc3722